interop_binaries/src/commands/janus_interop_aggregator.rs

```diff
@@ -130,17 +133,36 @@ async fn make_handler(
     datastore: Arc<Datastore<RealClock>>,
     dap_serving_prefix: String,
     aggregator_address: SocketAddr,
+    health_check_peers: Vec<Url>,
 ) -> anyhow::Result<impl Handler> {
     let keyring = Keyring::new();
 
     let upstream = format!("http://{aggregator_address}/").into_upstream();
     let proxy_handler = Proxy::new(
         Client::new(ClientConfig::default()).with_default_pool(),
-        upstream,
+        upstream.clone(),
     );
 
     let handler = Router::new()
-        .post("internal/test/ready", Json(serde_json::Map::new()))
+        .post("internal/test/ready", move |conn: Conn| {
+            let health_check_peers = health_check_peers.clone();
+            let client = Client::new(ClientConfig::default()).with_default_pool();
```

Comment:
We could save a bit of TCP connection setup if we constructed one client, then used a clone in the proxy, and a clone for the health check handler. It's not a big deal, since this is all test-only.